students/stuartkershaw/lesson03/strformat_lab.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+
+# Task One
+
+seq_in = (2, 123.4567, 10000, 12345.67)
+
+file_out = ''.join(['file_', format(seq_in[0], '03')])
+float_dec_out = format(seq_in[1], '.2f')
+sci_out = format(seq_in[2], '.2e')
+sci_sig_out = format(seq_in[3], '.2e')
+
+out_msg = '{}: {}, {}, {}'.format(file_out, float_dec_out, sci_out,
+                                  sci_sig_out)
+
+print(out_msg)
+
+# Task Two
+
+file_out_alt = f"file_{seq_in[0]:03}"
+float_dec_out_alt = f"{seq_in[1]:.2f}"
+sci_out_alt = f"{seq_in[2]:.2e}"
+sci_sig_out_alt = f"{seq_in[3]:.2e}"
+
+out_msg_alt = f"{file_out_alt}: {float_dec_out_alt}, {sci_out_alt}" \
+               "{sci_sig_out_alt}"
```

Comment:
Remember to add the `f` to the second line of the string or this will print out `{sci_sig_out_alt}`